source/utilities.tex

```diff
@@ -15739,6 +15847,7 @@
  needed when only explicit conversions are desired among the template arguments.
  \end{note} \\ \rowsep
 
+\indexlibrary{\idxcode{underlying_type}}%
```

Comment:
Looks like this might be a github bug? Some other views of this file show the file ending here!